### ISSUE TITLE
[Angular] has no exported  member 'ɵbl'.

### ISSUE THREAD

**Issue by sebitsi** [abpframework/abp#5360]
Hi. 

I have just downloaded latest 3.1.1 version of ABP.

`abp new Sebit.CIS -t app-pro -u angular`

after yarn and yarn start i got errors:

```
ERROR in node_modules/@volo/abp.ng.audit-logging/lib/services/entity-change.service.d.ts:17:32 - error TS2694: Namespace '"C:/Razvoj/Sebit/ABP/ABP Commercial/Ver3.1.1/angular/node_modules/@abp/ng.core/abp-ng.core"' has 
no exported member 'ɵbl'.

17     } & import("@abp/ng.core").ɵbl.PageQueryParams>): Observable<EntityChange.Response>;
                                  ~~~
node_modules/@volo/abp.ng.audit-logging/lib/components/entity-change/entity-changes.component.d.ts:18:41 - error TS2694: Namespace '"C:/Razvoj/Sebit/ABP/ABP Commercial/Ver3.1.1/angular/node_modules/@abp/ng.core/abp-ng.core"' has no exported member 'ɵbl'.

18     changeTypes: import("@abp/ng.core").ɵbl.Option<typeof eEntityChangeType>[];
                                           ~~~
node_modules/@volo/abp.ng.audit-logging/lib/states/audit-logging.state.d.ts:13:136 - error TS2694: Namespace '"C:/Razvoj/Sebit/ABP/ABP Commercial/Ver3.1.1/angular/node_modules/@abp/ng.core/abp-ng.core"' has no exported 
member 'ɵbl'.

13     get({ patchState }: StateContext<AuditLogging.State>, { payload }: GetAuditLogs): import("rxjs").Observable<import("@abp/ng.core").ɵbl.PagedResponse<AuditLogging.Log>>;
                                                                                                                                          ~~~
node_modules/@volo/abp.ng.saas/lib/states/saas.state.d.ts:16:133 - error TS2694: Namespace '"C:/Razvoj/Sebit/ABP/ABP Commercial/Ver3.1.1/angular/node_modules/@abp/ng.core/abp-ng.core"' has no exported member 'ɵbl'.     

16     getTenants({ patchState }: StateContext<Saas.State>, { payload }: GetTenants): import("rxjs").Observable<import("@abp/ng.core").ɵbl.PagedResponse<Saas.Tenant>>;
                                                                                                                                       ~~~
node_modules/@volo/abp.ng.saas/lib/states/saas.state.d.ts:20:135 - error TS2694: Namespace '"C:/Razvoj/Sebit/ABP/ABP Commercial/Ver3.1.1/angular/node_modules/@abp/ng.core/abp-ng.core"' has no exported member 'ɵbl'.     

20     getEditions({ patchState }: StateContext<Saas.State>, { payload }: GetEditions): import("rxjs").Observable<import("@abp/ng.core").ɵbl.PagedResponse<Saas.Edition>>;
                                                                                                                                         ~~~
node_modules/@volo/abp.ng.saas/lib/states/saas.state.d.ts:25:114 - error TS2694: Namespace '"C:/Razvoj/Sebit/ABP/ABP Commercial/Ver3.1.1/angular/node_modules/@abp/ng.core/abp-ng.core"' has no exported member 'ɵbl'.     

25     getLatestTenants({ patchState }: StateContext<Saas.State>): import("rxjs").Observable<import("@abp/ng.core").ɵbl.PagedResponse<Saas.Tenant>>;
                                                                                                                    ~~~
node_modules/@volo/abp.ng.identity/lib/components/users/users.component.d.ts:48:53 - error TS2694: Namespace '"C:/Razvoj/Sebit/ABP/ABP Commercial/Ver3.1.1/angular/node_modules/@abp/ng.core/abp-ng.core"' has no exported 
member 'ɵbl'.

48     lockDurationTypeOptions: import("@abp/ng.core").ɵbl.Option<typeof Identity.UserLockDurationType>[];
                                                       ~~~
node_modules/@volo/abp.ng.identity/lib/states/identity.state.d.ts:17:133 - error TS2694: Namespace '"C:/Razvoj/Sebit/ABP/ABP Commercial/Ver3.1.1/angular/node_modules/@abp/ng.core/abp-ng.core"' has no exported member 'ɵbl'.

17     getRoles({ patchState }: StateContext<Identity.State>, { payload }: GetRoles): import("rxjs").Observable<import("@abp/ng.core").ɵbl.PagedResponse<Identity.RoleItem>>;
                                                                                                                                       ~~~
node_modules/@volo/abp.ng.identity/lib/states/identity.state.d.ts:21:133 - error TS2694: Namespace '"C:/Razvoj/Sebit/ABP/ABP Commercial/Ver3.1.1/angular/node_modules/@abp/ng.core/abp-ng.core"' has no exported member 'ɵbl'.

21     getUsers({ patchState }: StateContext<Identity.State>, { payload }: GetUsers): import("rxjs").Observable<import("@abp/ng.core").ɵbl.PagedResponse<Identity.UserItem>>;
                                                                                                                                       ~~~
node_modules/@volo/abp.ng.identity/lib/states/identity.state.d.ts:27:143 - error TS2694: Namespace '"C:/Razvoj/Sebit/ABP/ABP Commercial/Ver3.1.1/angular/node_modules/@abp/ng.core/abp-ng.core"' has no exported member 'ɵbl'.

27     getClaimTypes({ patchState }: StateContext<Identity.State>, { payload }: GetClaimTypes): import("rxjs").Observable<import("@abp/ng.core").ɵbl.PagedResponse<Identity.ClaimType>>;
                                                                                                                                                 ~~~
node_modules/@volo/abp.ng.language-management/lib/states/language-management.state.d.ts:16:142 - error TS2694: Namespace '"C:/Razvoj/Sebit/ABP/ABP Commercial/Ver3.1.1/angular/node_modules/@abp/ng.core/abp-ng.core"' has 
no exported member 'ɵbl'.

16     get({ patchState }: StateContext<LanguageManagement.State>, { payload }: GetLanguages): import("rxjs").Observable<import("@abp/ng.core").ɵbl.PagedResponse<LanguageManagement.Language>>;
                                                                                                                                                ~~~
node_modules/@volo/abp.ng.language-management/lib/states/language-management.state.d.ts:21:159 - error TS2694: Namespace '"C:/Razvoj/Sebit/ABP/ABP Commercial/Ver3.1.1/angular/node_modules/@abp/ng.core/abp-ng.core"' has 
no exported member 'ɵbl'.

21     getLanguageTexts({ patchState }: StateContext<LanguageManagement.State>, { payload }: GetLanguageTexts): import("rxjs").Observable<import("@abp/ng.core").ɵbl.PagedResponse<LanguageManagement.LanguageText>>;      
                                                                                                                                                                 ~~~
node_modules/@volo/abp.ng.identity-server/lib/services/identity-server-state.service.d.ts:11:42 - error TS2694: Namespace '"C:/Razvoj/Sebit/ABP/ABP Commercial/Ver3.1.1/angular/node_modules/@abp/ng.core/abp-ng.core"' has no exported member 'ɵbl'.

11     getClients(): import("@abp/ng.core").ɵbl.PagedResponse<import("../models").IdentityServer.Client.Item>;
                                            ~~~
node_modules/@volo/abp.ng.identity-server/lib/states/identity-server.state.d.ts:14:163 - error TS2694: Namespace '"C:/Razvoj/Sebit/ABP/ABP Commercial/Ver3.1.1/angular/node_modules/@abp/ng.core/abp-ng.core"' has no exported member 'ɵbl'.

14     getIdentityResources({ patchState }: StateContext<IdentityServer.State>, { payload }: GetIdentityResources): import("rxjs").Observable<import("@abp/ng.core").ɵbl.PagedResponse<IdentityServer.IdentityResource>>;  
   [91m                                                                                                                                                                  ~~~
node_modules/@volo/abp.ng.identity-server/lib/states/identity-server.state.d.ts:19:153 - error TS2694: Namespace '"C:/Razvoj/Sebit/ABP/ABP Commercial/Ver3.1.1/angular/node_modules/@abp/ng.core/abp-ng.core"' has no exported member 'ɵbl'.

19     getApiResources({ patchState }: StateContext<IdentityServer.State>, { payload }: GetApiResources): import("rxjs").Observable<import("@abp/ng.core").ɵbl.PagedResponse<IdentityServer.ApiResource>>;
                                                                                                                                                           ~~~
node_modules/@volo/abp.ng.identity-server/lib/states/identity-server.state.d.ts:24:143 - error TS2694: Namespace '"C:/Razvoj/Sebit/ABP/ABP Commercial/Ver3.1.1/angular/node_modules/@abp/ng.core/abp-ng.core"' has no exported member 'ɵbl'.

24     getClients({ patchState }: StateContext<IdentityServer.State>, { payload }: GetClients): import("rxjs").Observable<import("@abp/ng.core").ɵbl.PagedResponse<IdentityServer.Client.Item>>;
                                                                                                                                                 ~~~

```

This is pure downloaded project without any modifications.
Just yarn and yarn start.

I'm using yarn version 1.22.4

Please advice.

**Comment by mehmet-erim** [comment]
Hi @sebitsi 

We are looking for the source of the error and we will fix the problem.

You can ignore the errors temporarily, open `tsconfig.base.json` and add the `"skipLibCheck": true` property to the `compilerOptions` object as shown below:

```diff
// tsconfig.base.json

"compilerOptions": {
     //...
+    "skipLibCheck": true,
  }
```

**Comment by sebitsi** [comment]
@mehmet-erim Thanks for quick response. This workaround is working.

Now i have another issue. 

My server has no account/login route anymore.
I mean it is in swagger. I can execute it as REST call with success.
But there is no page account/login anymore.

So if try to login from angular app i got

![image](https://user-images.githubusercontent.com/2194876/92753926-a5bec200-f38a-11ea-97a7-00a7859b6435.png)
 
Original server is forking perfect. Only mine is not.
What have you changed ? Is there any module missing or changed ?

Please advice.

**Comment by sebitsi** [comment]
Strange. I just made full (solution) rebuild.
Now it works....

**Comment by mehmet-erim** [comment]
The issue has been resolved with the v3.1.2 release. Thanks for reporting.
